src/sql/finalize.sql

```diff
@@ -241,11 +349,24 @@ CREATE OPERATOR <=> (
 	COMMUTATOR = <=>
 );
 
+CREATE OPERATOR <=> (
+	PROCEDURE = _vectors_svecf32_operator_cosine,
+	LEFTARG = svector,
+	RIGHTARG = svector,
+	COMMUTATOR = <=>
+);
+
 -- List of functions
 
 CREATE FUNCTION pgvectors_upgrade() RETURNS void
 IMMUTABLE STRICT PARALLEL SAFE LANGUAGE c AS 'MODULE_PATHNAME', '_vectors_pgvectors_upgrade_wrapper';
 
+CREATE FUNCTION svector_from_split_array("dims" INT, "index" INT[], "value" real[]) RETURNS svector
```

Comment:
```suggestion
CREATE FUNCTION svector_from_array("dims" INT, "index" INT[], "value" real[]) RETURNS svector
```

Reply:
And please check and force the index to be sorted (monotonically increasing), raise exception otherwise.

Reply:
The current implementation will sort arrays and check whether the lengths of two arrays are equal, whether either array contains null, and whether there are duplicate keys.

https://github.com/tensorchord/pgvecto.rs/blob/65b83f0c2f3f20d252ce1af640d171266813e50e/src/datatype/svecf32.rs#L709-L770

Reply:
LGTM